src/main/kotlin/com/terraformation/backend/accelerator/variables/VariableValuesHelper.kt

```diff
@@ -0,0 +1,125 @@
+package com.terraformation.backend.accelerator.variables
+
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.docprod.VariableId
+import com.terraformation.backend.documentproducer.model.AppendValueOperation
+import com.terraformation.backend.documentproducer.model.BaseVariableValueProperties
+import com.terraformation.backend.documentproducer.model.DeleteValueOperation
+import com.terraformation.backend.documentproducer.model.ExistingNumberValue
+import com.terraformation.backend.documentproducer.model.ExistingSelectValue
+import com.terraformation.backend.documentproducer.model.ExistingTextValue
+import com.terraformation.backend.documentproducer.model.ExistingValue
+import com.terraformation.backend.documentproducer.model.NewNumberValue
+import com.terraformation.backend.documentproducer.model.NewSelectValue
+import com.terraformation.backend.documentproducer.model.NewTextValue
+import com.terraformation.backend.documentproducer.model.NumberVariable
+import com.terraformation.backend.documentproducer.model.SelectVariable
+import com.terraformation.backend.documentproducer.model.TextVariable
+import com.terraformation.backend.documentproducer.model.UpdateValueOperation
+import com.terraformation.backend.documentproducer.model.ValueOperation
+import com.terraformation.backend.documentproducer.model.Variable
+import java.math.BigDecimal
+
+fun getNumberValue(values: Map<String, ExistingValue>, stableId: String): BigDecimal? {
```

Comment:
In the latest revision, this uses StableID enum as an input. But I agree with your comment, I can refactor this set of helpers into a class.

Reply:
In the latest revision, I left the helper function as is, but updated the variables to be named valuesByStableId, which should help with the clarity issues for now.

In the later PR, I want to take a bit of time to work through if we can simplified the "Read/Write typed variable values by stable ID" operations. It seems like we may be able to determine the type of values by the actual variable data type, instead of relying on us to pick the right helper method here.